browser/ui/brave_shields_data_controller.h

```diff
@@ -37,8 +39,10 @@ class BraveShieldsDataController
                          const std::string& subresource);
   void ClearAllResourcesList();
   int GetTotalBlockedCount();
-  bool GetIsBraveShieldsEnabled();
+  bool IsBraveShieldsEnabled();
```

Comment:
why rename this? naming convention change? If so, why not change the others?

Reply:
@bridiver pointed this out. It's either `Get` or `Is` not both.

Reply:
I would suggest `GetBraveShieldsEnabled` to match up with `brave_shields::GetBraveShieldsEnabled`, but just a nit